docs/site/Fields-filter.md

```diff
@@ -50,14 +47,18 @@ Return customer information only with `name` and `address`, and hide their `id`:
 await customerRepository.find({fields: {id: false, name: true, address: true}});
 ```
 
+Or use stringified JSON format:
+
+`/orders?filter={"fields":{"name":true,"address":true,"id":false}}`
+
 {% include code-caption.html content="REST" %}
 
-Include all required properties as a workaround:
+Include all required properties:
 `/customers?filter[fields][name]=true&filter[fields][address]=true`
 
-Or use stringified JSON format:
+Or omit the hidden property:
 
-`/orders?filter={"fields":{"name":true,"address":true,"id":false}}`
+`/customers?filter[fields][id]=false`
```

Comment:
Btw this might not be correct if the model has properties other than `id`, `name`, and `address`.